ydb/mvp/core/core_ydb.h

```diff
@@ -15,6 +15,7 @@
 #include <library/cpp/json/json_value.h>
 #include <util/generic/strbuf.h>
 #include <library/cpp/deprecated/atomic/atomic.h>
+#include "access_service_type.h"
```

Comment:
maybe it's better to include this header in cpp files only (where it's needed)?

Reply:
removed this header